### ISSUE TITLE
Sensor activation date

### ISSUE THREAD

**Issue by JotaMyke** [pachi81/GlucoDataHandler#199]
Hello
In class ReceiveData.kt how about getting the sensor activation date ?
Ex: in Java:
 Long sensorActivedDate = data_.getJSONArray("activeSensors").getJSONObject(0).getJSONObject("sensor").getLong("a");

Good new year

JotaMyke

**Comment by pachi81** [comment]
Yes it is already on my list, but not have a high prio at the moment.
I akso have to take care for other sources...
